Examples/Python/tests/test_examples.py

```diff
@@ -1173,7 +1173,15 @@ def test_full_chain_odd_example_pythia_geant4(tmp_path):
     env["ACTS_LOG_FAILURE_THRESHOLD"] = "ERROR"
     try:
         stdout = subprocess.check_output(
-            [sys.executable, str(script), "-n1", "--geant4", "--ttbar"],
+            [
+                sys.executable,
+                str(script),
+                "-n1",
+                "--geant4",
+                "--ttbar",
+                "--ttbar-pu",
+                "50",
```

Comment:
I made this explicit there @Corentin-Allaire

Reply:
Ah yes I missed that ! Then this is perfect !